listener_clients/gpo-listener/gpo-listener.py

```diff
@@ -262,7 +264,7 @@ def on_reassign(oldDeviceId, newDeviceId, gpoGroupId):
             gpo_group["deviceId"] = newDeviceId
 
     saveConfig()
-
+    sio.emit('listener_reassign_gpo', (gpoGroupId, oldDeviceId, newDeviceId));
```

Comment:
We forgot to document this in the code.
We are trying to deprecate the old `listener_reassign_gpo` events, migrating to a single event for all the different types of listeners. I'll update this and add some warnings in the codebase.

Reply:
Hi @MatteoGheza, ah no problem, that makes sense. Let me know if I can help test in any way. Part of the problem with the reassign was using the variable `listener_clients[I].gpoGroupId` in `index.ts`, which should have been `listener_clients[i].internalId`. I guess this is part of the migration to a single interface for all devices.